### PR TITLE
Grants: replace Age field with Group Age

### PR DESCRIPTION
**Pull request by estyxx**

### CODE REVIEW

File: backend/grants/admin.py [hunk 1]
```diff
@@ -13,7 +13,6 @@
     "name",
     "full_name",
     "email",
-    "age",
```
Comment (marcoacierno) [inline backend/grants/admin.py:16]: did you forget to add age_group here?

File: frontend/src/components/grant-form/index.tsx [hunk 2]
```diff
@@ -66,12 +65,12 @@ export const GrantForm = ({ conference }: Props) => {
       formState.setField("fullName", user.fullName);
       formState.setField("name", user.name);
       formState.setField("gender", user.gender);
-      if (user.dateBirth) {
-        formState.setField(
-          "age",
-          new Date().getFullYear() - new Date(user.dateBirth).getFullYear(),
-        );
-      }
+      // if (user.dateBirth) {
```
Comment (marcoacierno) [inline frontend/src/components/grant-form/index.tsx:68]: do we want to automatically pre-select the age group?

File: frontend/src/locale/index.ts [hunk 3]
```diff
@@ -354,7 +354,18 @@ export const messages = {
     "grants.form.fields.email": "Email",
     "grants.form.fields.email.description":
       "Here you will receive a notification about your application.",
-    "grants.form.fields.age": "Age",
+    "grants.form.fields.ageGroup": "Age",
+    "grants.form.fields.ageGroup.values.selectOption": "Select Option",
+    "grants.form.fields.ageGroup.values.range_less_than_10":
```
Comment (marcoacierno) [inline frontend/src/locale/index.ts:359]: is anyone ever going to select this :P 😂

File: backend/grants/models.py [hunk 4]
```diff
@@ -25,6 +25,16 @@
 
 
 class Grant(TimeStampedModel):
+    class AgeGroup(models.TextChoices):
+        RANGE_LESS_THAN_10 = "range_less_than_10", _("10 years old or under")
```
Comment (marcoacierno) [inline backend/grants/models.py:29]: I am not 100% convinced about having this "less than 10 years old" option, it is not like someone is going to use it 🤔 

I need to think about it a bit

File: frontend/src/locale/index.ts (hunk 3 above)
Comment (estyxx) [reply frontend/src/locale/index.ts:359]: Yes, at PyCon 11 we had one grant request for a 9 years old.

File: frontend/src/components/grant-form/index.tsx (hunk 2 above)
Comment (estyxx) [reply frontend/src/components/grant-form/index.tsx:68]: yes I was thinking about how to do it in an elegant way... :D

File: backend/grants/admin.py (hunk 1 above)
Comment (estyxx) [reply backend/grants/admin.py:16]: No, I will look at the export in another PR, as I need to remove some stuff now that we have the user in the model.

**Review by marcoacierno — APPROVED** [review]

**Pull request merged**